common/src/main/res/values/strings.xml

```diff
@@ -791,4 +791,5 @@
     <string name="sensor_update_notification">In order for the app to send sensor updates in the background a notification will be created during the update. You may use the button below to manage the appearance of this notification. It is recommended to minimize the notification to hide the icon.</string>
     <string name="sensor_worker_notification_channel">Manage Sensor Worker Notification</string>
     <string name="sensor_update_frequency_summary">Change the frequency of updates for sensors that do not update instantly.</string>
+    <string name="notification_channels_description">Notification channels allow you to control the look and feel of a certain type of notification. You can create notification channels on the fly by creating a notification with the channel parameter, refer to the help icon in the upper right hand corner to learn more. Below is a list of all notification channels created by your server admin or the app. Select the edit icon to adjust channel settings like ringtone and Do Not Disturb override. Select the delete icon to remove the channel from this list.\n\nDeleting a channel does not reset channel settings, if a notification is created with the same channel name it will retain its previous settings.\n\nWhile you may delete an app created notification channel, it will reappear once it is recreated.</string>
```

Comment:
Some text suggestions. Feel free to ignore them if you don't agree, I'm not a native English speaker.

> You can create notification channels on the fly by creating a notification with the channel parameter

"on the fly" sounds unnecessary to me, and you actually need to send a notification, not just write one.

> Below is a list of all notification channels created by your server admin or the app

"by your server admin" hasn't been used before, usually it's "your server" or "your instance" which sounds better to me. It's also a list of channels created by the server *and* app most of the time.

> if a notification is created with the same channel name it will retain its previous settings

The app never creates notifications, I'd use "received", and I think re-used sounds better than retained.

> While you may delete an app created notification channel, it will reappear once it is recreated.

This doesn't explain when it is recreated, up until now it has been suggested that the only way to recreate a channel is when sending a notification with the same `channel` parameter.

Reply:
Made some adjustments based on these recommendations :)